odoo_install.sh

```diff
@@ -150,8 +156,13 @@ if [ $GENERATE_RANDOM_PASSWORD = "True" ]; then
     OE_SUPERADMIN=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1)
 fi
 sudo su root -c "printf 'admin_passwd = ${OE_SUPERADMIN}\n' >> /etc/${OE_CONFIG}.conf"
-sudo su root -c "printf 'xmlrpc_port = ${OE_PORT}\n' >> /etc/${OE_CONFIG}.conf"
+if [ $OE_VERSION == "12.0" ];then
```

Comment:
Hmm, not sure why we should do this?

Reply:
If I remember correctly the xmlrpc port  conf was changed after 12 right? Anyways I've changed it to >=12.0